docs/proxy/datacenter_proxy/examples.md

```diff
@@ -0,0 +1,668 @@
+async function useProxy() {
+    const response = await got("https://api.apify.com/v2/browser-info", {
+        agent: {
+            http: new HttpProxyAgent(proxyUrl),
+            https: new HttpsProxyAgent(proxyUrl)
+        }
+    });
+    console.log(response.body)
+};
+
+useProxy();
+// Should return the same clientIp as
+useProxy();
+</marked-tab>
+
+
+<marked-tab header="Python 3+" lang="python">
+import urllib.request as request
+import ssl
+
+def do_request():
+    # Replace <YOUR_PROXY_PASSWORD> below with your password
+    # found at https://my.apify.com/proxy
+    password = "<YOUR_PROXY_PASSWORD>"
+    proxy_url = f"http://session-my_session:{password}@proxy.apify.com:8000"
+    proxy_handler = request.ProxyHandler({
+        "http": proxy_url,
+        "https": proxy_url,
+    })
+
+    ctx = ssl.create_default_context()
+    ctx.check_hostname = False
+    ctx.verify_mode = ssl.CERT_NONE
+    httpHandler = request.HTTPSHandler(context=ctx)
+
+    opener = request.build_opener(httpHandler,proxy_handler)
+    return opener.open("https://api.apify.com/v2/browser-info").read()
+
+print(do_request())
+print("Should return the same clientIp as ")
+print(do_request())
+</marked-tab>
+
+
+<marked-tab header="Python 2+" lang="python">
+import six
+from six.moves.urllib import request
+import ssl
+
+def do_request():
+    # Replace <YOUR_PROXY_PASSWORD> below with your password
+    # found at https://my.apify.com/proxy
+    password = "<YOUR_PROXY_PASSWORD>"
+    proxy_url = (
+        "http://session-my_session:%s@proxy.apify.com:8000" %
+        (password)
+    )
+    proxy_handler = request.ProxyHandler({
+        "http": proxy_url,
+        "https": proxy_url,
+    })
+
+    ctx = ssl.create_default_context()
+    ctx.check_hostname = False
+    ctx.verify_mode = ssl.CERT_NONE
+    httpHandler = request.HTTPSHandler(context=ctx)
+
+    opener = request.build_opener(httpHandler,proxy_handler)
+    return opener.open("https://api.apify.com/v2/browser-info").read()
+
+print(do_request())
+print("Should return the same clientIp as ")
+print(do_request())
+</marked-tab>
+
+
+<marked-tab header="PHP" lang="php">
+<?php
+function doRequest() {
+    $curl = curl_init("https://api.apify.com/v2/browser-info");
+    curl_setopt($curl, CURLOPT_RETURNTRANSFER, 1);
+    curl_setopt($curl, CURLOPT_PROXY, "http://proxy.apify.com:8000");
+    // Replace <YOUR_PROXY_PASSWORD> below with your password
+    // found at https://my.apify.com/proxy
+    curl_setopt($curl, CURLOPT_PROXYUSERPWD, "session-my_session:<YOUR_PROXY_PASSWORD>");
+    $response = curl_exec($curl);
+    curl_close($curl);
+    return $response;
+}
+$response1 = doRequest();
+$response2 = doRequest();
+echo $response1;
+echo "\nShould be contain same clientIp as\n";
+echo $response2;
+?>
+</marked-tab>
+```
+
+### [](#two-requests-with-different-ip-addresses) Two requests with different IP addresses
+
+The IP addresses are chosen from the `SHADER` and `BUYPROXIES94952` proxy groups.
```

Comment:
This seems to imply that you need to specify some proxy groups to get different IPs, but that's not the case.